src/edu/asu/plp/tool/prototype/Main.java

```diff
@@ -413,45 +463,843 @@ private ProjectExplorerTree createProjectTree()
 	private Parent createToolbar()
 	{
 		HBox toolbar = new HBox();
-		toolbar.setPadding(new Insets(0, 0, 0, 5));
+		toolbar.setPadding(new Insets(1.5, 0, 1, 5));
 		toolbar.setSpacing(5);
 		ObservableList<Node> buttons = toolbar.getChildren();
 		
 		EventHandler<MouseEvent> listener;
 		Node button;
 		
+		DropShadow lBlueShadow = new DropShadow();
+		lBlueShadow.setColor(Color.LIGHTBLUE);
+		DropShadow dBlueShadow = new DropShadow();
+		dBlueShadow.setColor(Color.DARKBLUE);
+		
 		// TODO: replace event handlers with actual content
+		Node projectButton = new ImageView("toolbar_new.png");
+		projectButton.addEventHandler(MouseEvent.MOUSE_ENTERED,
+				new EventHandler<MouseEvent>() {
+					@Override
+					public void handle(MouseEvent e)
+					{
+						projectButton.setEffect(lBlueShadow);
+					}
+				});
+		// Removing the shadow when the mouse cursor is off
+		projectButton.addEventHandler(MouseEvent.MOUSE_EXITED,
+				new EventHandler<MouseEvent>() {
+					@Override
+					public void handle(MouseEvent e)
+					{
+						projectButton.setEffect(null);
+					}
+				});
+		projectButton.addEventHandler(MouseEvent.MOUSE_PRESSED,
+				new EventHandler<MouseEvent>() {
+					@Override
+					public void handle(MouseEvent e)
+					{
+						console.println("New Project Clicked");
+						
+						createNewProject();
+						
+						projectButton.setEffect(dBlueShadow);
+					}
+				});
+		projectButton.addEventHandler(MouseEvent.MOUSE_RELEASED,
+				new EventHandler<MouseEvent>() {
+					@Override
+					public void handle(MouseEvent e)
+					{
+						projectButton.setEffect(lBlueShadow);
+					}
+				});
+		buttons.add(projectButton);
+		
 		button = new ImageView("toolbar_new.png");
 		listener = (event) -> console.println("new Project Clicked");
 		button.setOnMouseClicked(listener);
 		buttons.add(button);
 		
-		button = new ImageView("menu_new.png");
+		Node newFileButton = new ImageView("menu_new.png");
 		listener = (event) -> console.println("New File Clicked");
-		button.setOnMouseClicked(listener);
-		buttons.add(button);
+		newFileButton.setOnMouseClicked(listener);
+		buttons.add(newFileButton);
 		
 		button = new ImageView("toolbar_open.png");
 		listener = this::onOpenProjectClicked;
 		button.setOnMouseClicked(listener);
 		buttons.add(button);
 		
+		buttons.add(new Separator(Orientation.VERTICAL));
+		
 		button = new ImageView("toolbar_save.png");
-		listener = (event) -> console.println("Save Project Clicked");
+		listener = this::onSaveProjectClicked;
 		button.setOnMouseClicked(listener);
```

Comment:
This looks like something I did, but it might look better on one line as follows:
```
button.setOnMouseClicked(this::onSaveProjectClicked);
```